automobile/tools.md

```diff
@@ -1,8 +1,8 @@
 # Tools
 
-This section will present various tools to be used for automobile simulation sin
-Webots.
+This section will present various tools to be used for automobile simulations in Webots and a tutorial explaining step by step how to create a real scenario inspired from a real world map.
```

Comment:
We should avoid using the future tense when not needed.
will present &rarr; presents
real scenario &rarr; scenario
step by step &rarr; step-by-step